### ISSUE TITLE
Flatpak gnome 41 info/warning

### ISSUE THREAD

**Issue by roryst** [IBBoard/cawbird#492]
Using Fedora 37, when I try to upgrade my flatpaks, i receive this: 

Info: runtime org.gnome.Platform branch 41 is end-of-life, with reason:
   The GNOME 41 runtime is no longer supported as of September 17, 2022. Please ask your application developer to migrate to a supported platform.
Info: applications using this runtime:
   uk.co.ibboard.cawbird

Nothing to do.

**Comment by IBBoard** [comment]
This is why I don't like the Flatpak and Snap approach! Fedora 37 is still supported, but _we_ have to fix the dependencies.

I've updated the platform version 43 and assumed that will work because I couldn't find a list of supported platforms but Gnome 43 has been released. There are [test builds building](https://github.com/flathub/uk.co.ibboard.cawbird/pull/26). If those build okay then I can release an update in the next day or so.
